frontend/src/main/scala/quasar/data.scala

```diff
@@ -98,6 +98,9 @@ object Data {
   val _obj =
     Prism.partial[Data, ListMap[String, Data]] { case Data.Obj(m) => m } (Data.Obj(_))
 
+  def singletonObj(k: String, v: Data): Data =
```

Comment:
That very well might be useful, though does it have to be one or the other? This is useful for this PR as it is typed as `Data` and has less syntax (as you don't need to construct tuples). I'd wager there are plenty of places in code where we're also constructing singleton objects ;)